### ISSUE TITLE
ImportError for HelloWorldApp when running develop package

### ISSUE THREAD

**Issue by kdelwat** [hackebrot/cookiedozer#4]
I can't seem to get the template to work, running Arch Linux with Python version 3.5.2. I hope the following information is useful 😄.

# Steps to reproduce
+ Install the cookiecutter template with default options
+ Create a new virtual environment and install the dependencies locally
+ Run `python setup.py develop` as specified in the README
+ Run the `helloworld` executable

# Expected behaviour
The example app should launch.

# Actual behaviour
The executable `helloworld` fails with the following error (`~` replaces my home directory):

```python
Traceback (most recent call last):
  File "~/.venvs/helloworld/bin/helloworld", line 11, in <module>
    load_entry_point('helloworld', 'console_scripts', 'helloworld')()
  File "~/.venvs/helloworld/lib/python3.5/site-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "~/.venvs/helloworld/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "~/.venvs/helloworld/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "~/.venvs/helloworld/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "~/test/helloworld/helloworld/cli.py", line 9, in <module>
    from helloworld import HelloWorldApp
ImportError: cannot import name 'HelloWorldApp'
```

**Comment by hackebrot** [comment]
Hi @kdelwat! 👋 

Sorry to hear that. I haven't touched this project in ages and it worked just fine back then 😿 I don't have kivy set up and I'm not on Ubuntu/Manjaro Linux anymore.

I'm not sure how to help right now.

**Comment by hackebrot** [comment]
I've worked around the requirements and was able to reproduce the error you see. Can you please give it another try? I changed the import to be absolute so it would also work on Python 3.

**Comment by kdelwat** [comment]
The fix worked! HelloWorldApp is found. However, there seems to be a few issues with Python 3 support, including with gettext. I might try creating a PR at some point to resolve the issues.

**Comment by hackebrot** [comment]
Awesome, that would be great 😄

**Comment by hackebrot** [comment]
Resolved in #5
